tests/auth/login/login.spec.js

```diff
@@ -0,0 +1,175 @@
+import { expect, test } from "@playwright/test";
+
+// Function to generate random email (tried to do it via faker but didn't succeed, so googled this solution)
+const generateRandomString = () => Math.random().toString(36).substring(7);
```

Comment:
move it to separated directory(helpers, utils as example) and import to the test